### DISCUSSION TITLE
Value comparison

### DISCUSSION THREAD

**Discussion by trend-jason-x-lin** [SigmaHQ/sigma-specification#82] [Q&A] [answered]
Hi all,

Is there a way to compare column values with specific values such as EventId > 10?
I saw there is an aggregation with the comparison (count(UserName) by SourceWorkstation > 3) but I don't need aggregation.
Thanks!

**Comment by frack113** [comment]
Hi,
Only [pySigma](https://github.com/SigmaHQ/pySigma) support value comparison

**Reply by trend-jason-x-lin** [reply]
@frack113:
Could you share the sigma syntax and how pySigma is supported?
There is a discussion https://github.com/SigmaHQ/sigma/issues/218 here and the greater-than (gt) or less-than(lt) operator is not supported yet.

**Reply by frack113** [reply, marked as answer]
HI,
The implementation will be in the [sigma specification V2](https://github.com/SigmaHQ/sigma-specification/blob/version_2/Sigma_specification.md#modifier-types).
To writte "EventId > 10" it will be `EventId|gt: 10`.

BUT V2 is under construction, it can be (and will be) modified or updated.
